packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -47,6 +51,12 @@ const AppModals = ({
         case 'signup':
             ComponentToLoad = <AccountSignupModal />;
             break;
+        case 'request_email':
```

Comment:
after verifying the old email we send a email to the olld_email and after clicking that, the user is redirected to the request_email page which should change his email